tests/core/test_stellargraph.py

```diff
@@ -427,3 +428,69 @@ def test_feature_conversion_from_iterator():
     ab = gs.get_feature_for_nodes([4, 5], "B")
     assert ab.shape == (2, 10)
     assert ab[:, 0] == pytest.approx([4, 5])
+
+
+def example_benchmark_graph(feature_size=None, n_nodes=100, n_edges=200, n_types=4):
+    G = nx.Graph()
+
+    G.add_nodes_from(range(n_nodes))
+    edges = [
+        (random.randint(0, n_nodes - 1), random.randint(0, n_nodes - 1))
+        for _ in range(n_edges)
+    ]
+    G.add_edges_from(edges)
+
+    # Add example features
+    if feature_size is not None:
+        for v in G.nodes():
+            G.nodes[v]["feature"] = np.ones(feature_size)
+            G.nodes[v]["label"] = v % n_types
+
+    G = StellarGraph(G, node_features="feature")
```

Comment:
Does setting `node_features` make sense if `feature_size` is None? It is a wonder this doesn't break anything.

Reply:
I copied this from https://github.com/stellargraph/stellargraph/blob/f26f7733235573f5def19da3329fc1d92fdd71ee/tests/mapper/test_benchmark_generators.py#L27-L42

so apparently it doesn't break anything.

Reply:
Still conceptually wrong though. I'd rather see the 'correct' version that has an `else` clause using `StellarGraph(G)` when there are no features. No point propagating issues, even if there are previous usages.

Reply:
Fair enough.